lndmanage/lib/data_types.py

```diff
@@ -46,3 +46,18 @@ class NodeProperties:
     remote_base_fees: list
     remote_balances: list
     sent_received_per_week: int
+
+
+class KeyPair(tuple):
```

Comment:
Just noting that this may be a confusing naming due to the more common specific meaning of `key pair`.

Suggestions: `NodePair`, `PubkeyTuple`

Reply:
Had that thought as well, should have done that from the beginning, landed in rebase hell :grin: